test/test_ssa_direct_qmc.cc

```diff
@@ -100,5 +121,22 @@ TEST(SsaDistribution, MomentTest) {
             
     EXPECT_TRUE(KH_mu1) << os1.str();
     EXPECT_TRUE(KH_mu2) << os2.str();
+*/
+    constexpr double V_f_mu1 = n_proc;
+    constexpr double V_f_mu2 = n_proc*n_proc;
+    constexpr size_t n_events = 1000*1000;
+    try {
+        KHineq::kh_test(ssa, [](size_t j){return j;}, V_f_mu1, exact_mu1, n_events);
+    }
+    catch (std::runtime_error err) {
+        FAIL() << err.what();
+    }
+    try {
+        KHineq::kh_test(ssa, [](size_t j){return j*j;}, V_f_mu2, exact_mu2, n_events);
+    }
```

Comment:
That's correct! But it is also true of the original lines: if the first test fails, it will invoke Google Test's `FAIL` macro and stop the test. I think this behaviour is okay in a test; that said, if you'd prefer to run both tests even if one fails, you could use the `ADD_FAILURE` macro instead.